src/queries/comment/circle/pinnedBroadcast.ts

```diff
@@ -14,7 +14,7 @@ const resolver: CircleToPinnedBroadcastResolver = async (
       targetId: id,
       type: COMMENT_TYPE.circleBroadcast,
     },
-    orderBy: [{ column: 'created_at', order: 'desc' }],
+    orderBy: [{ column: 'updated_at', order: 'desc' }],
```

Comment:
If we use `updated_at` to order, the whenever a comment is edited it will be ranked first. Is that intended? 🤔